packages/dev/core/src/Cameras/arcRotateCamera.ts

```diff
@@ -1310,6 +1312,30 @@ export class ArcRotateCamera extends TargetCamera {
         super._updateRigCameras();
     }
 
+    /**
+     * @internal
+     */
+    public _calculateLowerRadiusFromModelBoundingSphere(minimumWorld: Vector3, maximumWorld: Vector3, radiusScale: number = 1): number {
+        const size = maximumWorld.subtract(minimumWorld);

```

Comment:
We can use a cached Vector2 here to avoid more GC

Reply:
I replaced the subtract and length calls with `Vector3.Distance` and that removed the need to create a vector.